### PR TITLE
chore(apache-maven): remove Nexus and links to it

### PR DESCRIPTION
**Pull request by Langleu**
related to the Nexus URL removal.

### CODE REVIEW

File: get-started/content/apache-maven.md [hunk 1]
```diff
@@ -149,108 +147,6 @@ An alternative is to directly use the new URL of Artifactory.
...
-    <url>
-      https://artifacts.camunda.com/artifactory/public/
-    </url>
-  </repository>
-</repositories>
-```
-
-### Enterprise Edition
-
 The `private` URL is a virtual repository, which aggregates a multitude of repositories.
```
Comment (garima-camunda) [inline get-started/content/apache-maven.md:150]: @Langleu Can we specify exactly which "private" URL we are referring to here. Can we consider mentioning either complete URL or link to a private URL?

File: get-started/content/apache-maven.md [hunk 2]
```diff
@@ -149,108 +147,6 @@ An alternative is to directly use the new URL of Artifactory.
...
 Those include customer enterprise artifacts and all public artifacts from `public` to make builds easier if those dependencies are needed.
```
Comment (garima-camunda) [inline get-started/content/apache-maven.md:151]: @Langleu The same as above. Can we specify exactly which "public" URL we are referring to here. Can we consider mentioning either complete URL or link to a public URL?

File: get-started/content/apache-maven.md (hunk 1 above)
Comment (Langleu) [reply get-started/content/apache-maven.md:150]: sure we can link it but it's already in the code example directly below it.

Comment (Langleu) [reply get-started/content/apache-maven.md]: linked the url

File: get-started/content/apache-maven.md (hunk 2 above)
Comment (Langleu) [reply get-started/content/apache-maven.md]: linked the url

File: get-started/content/apache-maven.md [hunk 3]
```diff
@@ -149,110 +147,8 @@ An alternative is to directly use the new URL of Artifactory.
...
-  </repository>
-</repositories>
-```
-
-### Enterprise Edition
-
-The `private` URL is a virtual repository, which aggregates a multitude of repositories.
-Those include customer enterprise artifacts and all public artifacts from `public` to make builds easier if those dependencies are needed.
+The [private](https://artifacts.camunda.com/artifactory/private/) URL is a virtual repository, which aggregates a multitude of repositories.
+Those include customer enterprise artifacts and all public artifacts from [public](https://artifacts.camunda.com/artifactory/public/) to make builds easier if those dependencies are needed.
```
Comment (ThorbenLindhauer) [inline get-started/content/apache-maven.md:151]: You can talk directly to the reader.

```suggestion
Those include customer enterprise artifacts and all public artifacts from [public](https://artifacts.camunda.com/artifactory/public/) to make builds easier if you need those dependencies.
```

File: get-started/content/apache-maven.md [hunk 4]
```diff
@@ -312,29 +208,33 @@ repositories {
...
-### Known issue with Artifactory
-
-Please find below the known issue in Artifactory while configuring Nexus repository as a Maven remote proxy.
-
-While testing the connection in Artifactory, if you get "Connection failed: Error 404: Not Found" message then you can ignore it, and continue to add the repository. The maven build should be able to download dependencies without any issues.
-
-### Browse Camunda Nexus
-In order to browse the Camunda Nexus artifacts, here are the links which can be used.
+### Browse Camunda Artifact Storage
+In order to browse the Camunda artifacts, here are the links which can be used.
```
Comment (ThorbenLindhauer) [inline get-started/content/apache-maven.md:212]:
```suggestion
In order to browse the Camunda artifacts, here are the links which you can use.
```

**Review by ThorbenLindhauer — CHANGES_REQUESTED** [review]
I don't understand the section `Additional config if used in conjunction with the camunda-bpm-ee URL.` in this version (also not in the previous version). Under which circumstances do readers need to apply this extra configuration?

**Comment by Langleu** [comment]
@ThorbenLindhauer, that part isn't required anymore and removed it.
It was related to a workaround always to send auth even if not asked. We resolved that at the beginning of a year with JFrog together.

**Review by ThorbenLindhauer — APPROVED** [review]
Thanks for clearing that up. I'll go ahead with merging and publishing the changes.

**Pull request merged**